### DISCUSSION TITLE
PTT audio delay?

### DISCUSSION THREAD

**Discussion by daveError** [allebb/pirrot#41] [Q&A] [answered]
Hello there,

I was wondering if there was any way to adjust the amount of time between pirrot tripping the PTT relay and the audio playing. I just want to make sure the beginning of the transmission doesn't get cut off, that's all.

If not, then perhaps this is a feature request. Close relay -> Wait half a second -> Play audio -> Open relay.

Thanks,
-Dave

**Comment by allebb** [comment]
Hi Dave,

Please give me a couple of days and I'll look into it (just a bit tied up at the moment) :) - I'll  work on an additional option to allow you to add a custom "wait" time before tripping the PTT relay 👍 

Cheers,
Bobby

**Comment by tekk** [comment, marked as answer]
Why would you use relays and not MOSFETs?
Yes, relays have advantages in terms of interference.
But as long your antenna is far enough away from the whole system, it works
fine.

**Comment by allebb** [comment]
Hi @daveError

Sorry for the long delay on this, I got completely tied up on other things... 

Just wanted to check if you have already found a solution, have moved on (no longer using Pirrot) or, if you would like me to look at, and implement a solution for your specific use case - If so, I should be able to get this done and released this week!

Cheers,
Bobby

**Reply by daveError** [reply]
Hey @allebb,

I shelved my little project over the past few weeks so no worries about timing.

I would still like to see the feature added at some point in the future if possible. All I want is an adjustable delay period preceding playback and idents. Presently, the relay puts me in no better a position than just putting the radio into VOX mode, so if pirrot could trip the relay and then wait a short period before playback, that would solve my issue.

Thank you for checking up on this,

-Dave

**Reply by allebb** [reply]
Ok, no worries - I understand @daveError.

I'll look at adding this as an option regardless 👍
